src/main/java/at/hannibal2/skyhanni/test/SkyHanniDebugsAndTests.kt

```diff
@@ -337,9 +337,9 @@ class SkyHanniDebugsAndTests {
 
         fun copyLocation(args: Array<String>) {
             val location = LocationUtils.playerLocation()
-            val x = LorenzUtils.formatDouble(location.x + 0.001).replace(",", ".")
-            val y = LorenzUtils.formatDouble(location.y + 0.001).replace(",", ".")
-            val z = LorenzUtils.formatDouble(location.z + 0.001).replace(",", ".")
```

Comment:
I have the feeling the replace call was not unnecessary

Reply:
Thought so to just didn't want to somehow have a problem if it was